bin/merged_prs.sc

```diff
@@ -50,6 +73,103 @@ def main(
       s"""|- ${pr.getTitle()}
           |  [\\#${pr.getNumber()}](${pr.getHtmlUrl()})
           |  ([$login](https://github.com/$login))""".stripMargin
-    println(formattedPR)
+    mergedPRs += formattedPR
   }
+
+  val releaseNotes =
+    template(
+      author,
+      firstTag,
+      codename,
+      lastTag,
+      mergedPRs.toList,
+      commits,
+      contributors
+    )
+
+  println(releaseNotes)
+}
+
+def today: String = {
+  val formatter = new SimpleDateFormat("yyyy-mm-dd");
+  formatter.format(new Date());
+}
+
+def template(
```

Comment:
I added the template here, so anyone can just run `amm bin/merged_prs v0.9.4 v0.9.5 > new_file.md`

Not everything is calculated, but I added anything that can be said for sure.

Reply:
Sort of a nit, but if it's expected for a user to use it this way and direct it to a file, then I'd just use the Ammonite `write(<target>, <data>)` to write it to disk as part of the script. Not super important though.

Reply:
Yeah, I was thinking about it, but was worried I was doing too many assumptions. I will add it.

Reply:
Added!